pkgs/development/haskell-modules/configuration-ghc-8.8.x.nix

```diff
@@ -123,11 +126,7 @@ self: super: {
   liquidhaskell = markBroken super.liquidhaskell;
 
   # This became a core library in ghc 8.10., so we don‘t have an "exception" attribute anymore.
-  exceptions = super.exceptions_0_10_5;
-
-  # ghc versions which don‘t match the ghc-lib-parser-ex version need the
-  # additional dependency to compile successfully.
-  ghc-lib-parser-ex = addBuildDepend self.ghc-lib-parser super.ghc-lib-parser-ex;
+  exceptions = super.exceptions_0_10_7;
 
   ormolu = super.ormolu_0_2_0_0;
```

Comment:
@cdepillabout Do you think we should switch the ormolu version to 0.5.0.1 instead ?

Reply:
I guess my comment is wrong since this is for ghc-8.8.x and for ghc-9.2.x, it should take the ormolu from the lts resolver.